<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Yes_No_Window</class>
 <widget class="QDialog" name="Yes_No_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="Yes">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Map File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="No">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Stadiums File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 italic 20pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Yes_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Souvenirs File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 16pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="Team_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Enter New Team Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Enter New Phone#:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stadium_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Enter New Stadium Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
   <property name="text">
    <string>Enter New Capacity:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 14pt &quot;Avenir&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
